src/validation.h

```diff
@@ -865,6 +873,22 @@ class ChainstateManager
     //! Get all chainstates currently being used.
     std::vector<CChainState*> GetAll();
 
+    //! Construct and activate a Chainstate on the basis of UTXO snapshot data.
+    //!
+    //! Steps:
+    //!
+    //! - Initialize an unused CChainState.
+    //! - Load its `CoinsViews` contents from `coins_file`.
+    //! - Verify that the hash of the resulting coinsdb matches the expected hash
+    //!   per assumeutxo chain parameters.
+    //! - Wait for our headers chain to include the base block of the snapshot.
```

Comment:
03e0de1

You should swap those two steps to be in-order ? Also mentions the `GetUTXOStats` one.